ui/README.md

```diff
@@ -39,12 +39,6 @@ You will need the following things properly installed on your computer.
 * [Yarn](https://yarnpkg.com/)
 * [Ember CLI](https://cli.emberjs.com/release/)
 * [Google Chrome](https://google.com/chrome/)
-- [lint-staged\*](https://www.npmjs.com/package/lint-staged)
-
-\* lint-staged is an optional dependency - running `yarn` will install it.
-If don't want optional dependencies installed you can run `yarn --ignore-optional`. If you've ignored the optional deps
-previously and want to install them, you have to tell yarn to refetch all deps by
-running `yarn --force`.
```

Comment:
Since this is no longer relevant I decided to remove it entirely but I'm open to replacing it with something different. I'm not sure if it's worth pointing out that it's not required for to build but or not?

https://yarnpkg.com/configuration/manifest#optionalDependencies

Reply:
I wonder if we should remove the optional dependencies and move them to devDeps (or maybe remove completely? Looks like at least one might have been leftover from when we had storybook). 

I'd like to avoid bloating the size of the UI assets with this change, if we can!

Reply:
I agree it doesn't seem relevant for us to use the `optionalDependencies` field anymore. It doesn't look like we use `babel-loader` and core shouldn't be necessary since we have `ember-cli-babel` so I will remove those and move `lint-staged` over to `devDependencies`.